autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -41,8 +41,7 @@ class Output(BlockSchema):
     def __init__(self):
         super().__init__(
             id="1ff065e9-88e8-4358-9d82-8dc91f622ba9",
-            description="This block forwards the `input` pin to `output` pin. "
-            "This block output will be static, the output can be consumed many times.",
+            description="Forwards the `input` to `output`, providing static output that can be reused.",
```

Comment:
This still makes very little sense to a normal person 

try replacing static with something like kept output?

Reply:
Updated